tests/unit/test_row_data.py

```diff
@@ -902,7 +905,10 @@ def test_RRRM_build_updated_request(rrrm_data):
     result = request_manager.build_updated_request()
 
     expected_result = _ReadRowsRequestPB(
-        table_name=TABLE_NAME, filter=row_filter.to_pb(), rows_limit=6
+        table_name=TABLE_NAME,
```

Comment:
is it easy to also check this in the system tests?

Reply:
I just ran the `system` test suite, but the ``build_updated_request`` doesn't seem to be covered there currently.